apps/docs/guide/deploying/validator.md

```diff
@@ -10,50 +10,141 @@ The following diagram illustrates the deployment system implemented by Sunodo.
+The `sunodo-validator` CLI provides to `start` command to execute a node:
+
+```shell
+sunodo-validator start
+    --chain-id=<option> (env: CARTESI_BLOCKCHAIN_ID)
+    --rpc-url=<value> (env: CARTESI_BLOCKCHAIN_HTTP_ENDPOINT)
+    --ws-url=<value> (env: CARTESI_BLOCKCHAIN_WS_ENDPOINT)
+    --private-key=<value>
+    --mnemonic-passphrase=<value> (env: CARTESI_AUTH_MNEMONIC)
+    --mnemonic-index=<value>
+    --kms-key-id=<value>
+    --validator=true|false
+    --verbose=true|false (env: CARTESI_LOG_LEVEL)
+    --embedded-database=true|false
+    --epoch-duration=<value> (env: CARTESI_EPOCH_DURATION)
+    --postgres-url=<value> (env: CARTESI_POSTGRES_ENDPOINT)
+    --snapshot-dir=<value> (env: CARTESI_SNAPSHOT_DIR)
+    --consensus=<value> (env: CARTESI_CONTRACTS_AUTHORITY_ADDRESS)
+    --history=<value> (env: CARTESI_CONTRACTS_HISTORY_ADDRESS)
+    --provider=<value> (env: SUNODO_CONTRACTS_PROVIDER)
+```
+
+The node requires the following external components:
+
+-   an ethereum provider for the selected base layer;
+-   a postgres database;
+-   the wallet that owns the `Authority` contract created in the previous section;
+
+### Ethereum provider
+
+The node requires a connection to the selected [base layer](./supported-networks.md) through a JSON-RPC provider.
+The provider must support both http and websocket.
+
+The user can run his own base layer node, or choose a third-party provider from the non exaustive list below:
+
+-   [Alchemy](http://alchemy.com)
+-   [Infura](https://infura.io)
+-   [Chainstack](http://chainstack.com)
+-   [GetBlock](https://getblock.io)
 
-::: info
-Add more information about wallet setup
+### Database
+
+The node uses a [postgres](http://postgresql.org) database to store application state (inputs and outputs), and serve those over a GraphQL API. The user must [setup a database server](https://www.postgresql.org/download/), or use a third-party managed database provider.
+
+The database credentials provided to the Sunodo node must have permissions to create new databases, because each application will have its own independent database. The schema of the application database does not need to be created beforehand because it's dynamically created by the Sunodo node.
+
+The `sunodo-validator start` command provides the option `--embedded-database` that will spin up a postgres instance. This option must only be used for test purposes, because persistence of data is turned off, there is no backup mechanism, and the security of the database access is not production-grade.
+
+### Wallet
+
+Sunodo validator node interacts with the `Authority` smart contract created above to submit claims of the application state once every epoch. In order to do that it needs to be configured with the base layer wallet that is the owner of the `Authority` contract.
+
+::: warning
+A wallet must be used by a single Sunodo node. Do not share the wallet with other applications or other Sunodo nodes as the nonce management may cause issues.
```

Comment:
```suggestion
The node uses a [postgres](http://postgresql.org) database to store application state (inputs and outputs) and serves those over a GraphQL API. The user must [setup a database server](https://www.postgresql.org/download/), or use a third-party managed database provider.

The database credentials provided to the Sunodo node must have permission to create new databases because each application will have its own independent database. The schema of the application database does not need to be created beforehand because it's dynamically created by the Sunodo node.

The `sunodo-validator start` command provides the option `--embedded-database` that will spin up a postgres instance. This option must only be used for test purposes because the persistence of data is turned off, there is no backup mechanism, and the security of the database access is not production-grade.

### Wallet

Sunodo validator node interacts with the `Authority` smart contract created above to submit claims of the application state once every epoch. To do that it needs to be configured with the base layer wallet that is the owner of the `Authority` contract.

::: warning
A wallet must be used by a single Sunodo node. Do not share the wallet with other applications or other Sunodo nodes as the nonce management may cause issues.
```